jfix-stdlib-serialization/src/main/java/ru/fix/stdlib/serialization/json/JsonObject.java

```diff
@@ -0,0 +1,30 @@
+package ru.fix.stdlib.serialization.json;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public abstract class JsonObject {
+
+    /**
+     * Other fields which processor will ignore
+     */
+    @JsonIgnore
+    private final Map<String, Object> otherFields = new HashMap<>();
```

Comment:
Let's name fields that are not explicitly defined in pojo as `implicit`. 
Since `implicit` is better explains  the purpose of this fields as implicitly serialized/deserialized by jackson.

```
@JsonIgnore
private final Map<String, Object> implicitFields = new HashMap<>();
@JsonAnyGetter
public Map<String, Object> implicitFields() {...}
@JsonAnySetter
 public void setImplicitField(String name, Object value) {...}
```

Reply:
done